### DISCUSSION TITLE
钉钉通知中增加@提醒人功能

### DISCUSSION THREAD

**Discussion by sh-hanson** [hhyo/Archery#1951] [Ideas]
看文档中，没有提到钉钉通知@指定人的功能。
比如提交工单后，到下一级审核人，钉钉通知的同时，再加上@审核人，加以提醒。
通常审核人都是主管一级，工作群比较多，很多时间消息都被淹没，如果有个@提醒就比较醒目。
不知道是否能实现，谢谢！

**Comment by nick2wang** [comment]
实现是可以的，更进一步可以为审批人创建带DING的钉钉待办，参考：https://github.com/nick2wang/Archery/commit/cae470a71431165519ecd3342b896bed5d1c6850
![image](https://user-images.githubusercontent.com/33473924/199701773-f30b5356-527e-40f5-9b18-f74abed44aae.png)

**Reply by sh-hanson** [reply]
钉钉待办倒是没有使用，看起来没有直接群里@，来的直接方便。因为大部分都是工作群里@人，也符合大家使用习惯。还是希望能有这个功能。

顺便请问钉钉待办功能在最新的docker部署中已经实现了吗？还是说需要自己二开啊？
